### ISSUE TITLE
Add CI Support

### ISSUE THREAD

**Issue by line72** [line72/montclair-pwa-ios#2]

**Comment by line72** [comment]
It looks like we need to run the following commands

```
npm install

xcodebuild -scheme Montclair -workspace platforms/ios/Montclair.xcworkspace -destination generic/platform=iOS build

xcodebuild -scheme Montclair -workspace platforms/ios/Montclair.xcworkspace -sdk iphoneos -configuration AppStoreDistribution archive -archivePath $PWD/build/Montclair.xcarchive

xcodebuild -exportArchive -archivePath /Users/dillavou/projects/montclair/cities/steamboat/montclair-steamboat-pwa-ios/upstream/build/Montclair.xcarchive/ -exportOptionsPlist exportOptions.plist -exportPath $PWD/build/
```

And we need an exportOptions.plist that looks something like:

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>method</key>
    <string>app-store</string>
    <key>teamID</key>
    <string>10-digit team id</string>
</dict>
</plist>
```

https://medium.com/xcblog/xcodebuild-deploy-ios-app-from-command-line-c6defff0d8b8

**Comment by line72** [comment]
Merged in e40e5d6dec6141f50586184da245e1a9fa427100
